WORKSPACE

```diff
@@ -36,6 +32,22 @@ load(
 
 container_repositories()
 
+load("@io_bazel_rules_go//go:def.bzl", "go_register_toolchains", "go_rules_dependencies")
+
+go_rules_dependencies()
```

Comment:
It is a pity that we couldn't call this repository rule in our repository rule... Maybe we can update README to indicate how should import bazel-toolchains repo in WORKSPACE as well?

Reply:
Yea. good idea. I forgot to do that. Will do.

Reply:
Updated the instructions in the docker_config.bzl, which is where the instructions located before.